addon/components/menu/button.hbs

```diff
@@ -1,3 +1,4 @@
+{{! template-lint-disable no-down-event-binding }}
```

Comment:
these are unrelated to typescript, but as I was debugging prettier warnings (which are reported as errors? 🤷 (I'm not used to the prettier CLI)), I decided to just take care of all warnings